### ISSUE TITLE
Humidity values not being sent by InfluxDB Datapush

### ISSUE THREAD

**Issue by GJRos** [domoticz/domoticz#1656]
Domoticz Ver: 3.8015 on Raspberry Pi

After a recent Beta update, the InfluxDB datapush is incorrectly sending all Humidity values as zero. It is happening with my RFLink and ESPEasy devices. 

2017-06-29 15:05:30.404 InfluxLink: value Humidity,idx=52,name=Room value=0
2017-06-29 15:05:30.404 InfluxLink: value Barometer,idx=52,name=Room value=846.0
2017-06-29 15:05:40.002 MQTT: Topic: domoticz/in, Message: {"idx":52,"nvalue":0,"svalue":"19.53;23.79;0;846.04;0"}
2017-06-29 15:05:40.045 InfluxLink: value Temperature,idx=52,name=Room value=19.5
2017-06-29 15:05:40.045 InfluxLink: value Humidity,idx=52,name=Room value=0

**Comment by Tommit1** [comment]
Could it be because or before of this request? 
#1648 
Now we have Version 3.8025!
Is Version 3.8015 before or after this correction?

**Comment by ikkemaniac** [comment]
@GJRos, please provide the build hash, you can get this info from the Setup -> About screen of your current build that fails.
Please test if #1648 causes this issue by reverting it (`git revert f24fa115bff3cd6948b2ad306ed14e35f6645483` and rebuild) while on the branch/commit you've built 3.8015 with.

@Tommit1 could be since the change is in the generic file. I've made this change because the true 'Humidity' sensor as a dummy device and the reference in the wiki (https://www.domoticz.com/wiki/Domoticz_API/JSON_URL%27s#Humidity) have levels saved as the nValue, humidity status ie 'comfort' levels are stored as sValue.

btw i  don't understand why the legacy svn version numbering is used. Life would be easier if everything was in commit hashes only.

**Comment by ikkemaniac** [comment]
Just tested on my own system with build hash 37b5133 (Version: 3.8025)
- Http Datapush functions correct. 
- InfluxDB: I can't test InfluxDB itself as i don't have it installed but this is the raw data captured that domoticz sends out when I set the device to a new value of 100:

```
POST /write?db=test HTTP/1.1
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/601.37 (KHTML, like Gecko) Chrome/54.0.40370.0 Safari/601.36
Host: localhost:12345
Accept: */*
Accept-Encoding: deflate, gzip
Content-Length: 38
Content-Type: application/x-www-form-urlencoded
Humidity,idx=1,name=humidity value=100
```
and Domoticz log:
`2017-06-29 20:32:08.772  InfluxLink: value Humidity,idx=1,name=humidity value=100`

So all that looks correct.

@GJRos are you sure you've selected humidity value and not humidity status in the 'value to send' dropdown?

**Comment by ikkemaniac** [comment]
I dug a little deeper. In @GJRos's post we see from the MQTT message that the nValue is empty, this looked strange to me as we should have the humidity value there per the wiki.

(everything below is on 37b51336)

If we look at [main/RFXtrx.h#L865](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/main/RFXtrx.h#L865)  downwards to [main/RFXtrx.h#L898](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/main/RFXtrx.h#L898) we different `pType` 's defined that can contain humidity data. 

In [main/mainworker.cpp#L2231](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/main/mainworker.cpp#L2231) to [main/mainworker.cpp#L2248](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/main/mainworker.cpp#L2248) sensor data is stored based on sensor type. We see different `decode_xxx` functions being called. Inside those functions data is saved to the db by calling `CSQLHelper::UpdateValue` in [main/SQLHelper.cpp#3182](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/main/SQLHelper.cpp#L3182) to [main/SQLHelper.cpp#3192](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/main/SQLHelper.cpp#L3192).

Only `pTypeHum` (dummy type device) eventually ends up saving it's humidity level data in nValue, all other `pType_xx_Hum_xx` devices end up saving the data in a concatenated string in sValue.

The root cause for this issue, or actually the cause which i thought #1648 patched, is that humidity is saved both in nValue and sValue while the datapush `module` makes no distinction and just assumes nValue (all per the way data is processed and saved in mainworker)

I have no quick fix for this as the `pType_xxx` is not currently available in the [push/BasePush.cpp CBasePush::ProcessSendValue](https://github.com/domoticz/domoticz/blob/37b51336d3ab8804244f9d4633c6c0594fdd7457/push/BasePush.cpp#L474) function.

Best thing to do might be to revert #1648 and just update the wiki with this as a known issue.

Best fix would be to change the way `pTypeHum` is saved (so save it in a concatenated string in sValue) to align it with the other types .

**Comment by GJRos** [comment]
My Build Hash is: 5d708b3
Compile Date: 2017-06-26 17:05:15

The issue started as soon as I did an update through Domoticz on 27/06/2017. I have checked for updates since but it shows there is no update available. 

I did check, I do have the Humidity Value selected and not the Humidity Status. I see in the forum that there are other also that have the same issue. 

@ikkemaniac I am not sure how to rebuild after reverting #1648.

**Comment by arbenik** [comment]
Version: 3.8025
Build Hash: 37b51336
Compile Date: 2017-06-28 06:32:47


I can tell that this issue started with Domoticz Ver: 3.8015 and still persisting since.
And it is not exclusive to InfluxDB Datapush but to HTTP Datapush also,

**Comment by ikkemaniac** [comment]
@GJRos How do you normally upgrade? I assumed you build it yourself; so after the revert you would be ready to run your cmake and make commands. If you use the built in update I’m afraid you have to wait for a new automatic build with a revert or fix in it. I've pushed a revert commit just now.

@arbenik see my post above, the issue is with the inconstancy in saving data vs parsing data in the datapush 'module' and is indeed unrelated to which type of datapush is used.

**Comment by GJRos** [comment]
@ikkemaniac I use the built in update. I will try and do the build myself.

**Comment by ikkemaniac** [comment]
@GJRos ok, let me know if you need help.

**Comment by GJRos** [comment]
@ikkemaniac I did try and check how to do it, but am struggling. Could you help me with the steps that I should do?

**Comment by ikkemaniac** [comment]
what system are you on? Linux?
I'm only familiar with Linux and the wiki is pretty good. Follow everything except chapter 3,4 and 5 as you already have domoticz running.

Basic thing to do is
- if you need `zwave`, download zwave library and build it
- if you need `Tellstick` support, download and install `libtelldus`
- install `cmake` program if you don't have it already. Cmake will make configuring the Domoticz source easy.
- configure domoticz by executing the `cmake` command with the correct parameters (see below)
- compile/build domiticz by executing the `make` command

I've ran into issues with the `libboost` dependencies (got errors when executing `cmake`) as they didn't match domoticz' requirements. The easiest fix for that was to go the `libboost` website, download the correct version (>1.62.00 I believe but the error should tell you), unpack in a directory somewhere, follow their readme file to build it (DON'T install it) and add a reference when you call `cmake` to tell it to use that version, i changed my `cmake` line from 
` cmake -DCMAKE_BUILD_TYPE=Release .`
to ` cmake -DCMAKE_BUILD_TYPE=Release . -DBoost_NO_SYSTEM_PATHS=ON -DBOOST_INCLUDEDIR=/my/path/to/lib/boost_1_62_0/`
which
- tells cmake to ignore the system libboost library
- points cmake to the libboost library location which it should use

Dont forget to build the 'zwave support' before you run the cmake command (it will throw an error and tell you, you won't break anything). For zwave it's really important that the zwave folder is called `open-zwave-read-only` and that it's in the same folder as where the domoticz folder is that holds the source. Carefully look at the directory sample that is shown in the wiki, your structure has to match that exactly.

So basically follow wiki, read error, correct and retry :D 

*ps: with a lot of retries, errors and trying agian the cmake files can get mangled up in my experience. I usually throw in a `git clean -xfd` (be carefull and read up on what this does!) before i execute another `cmake` command.

**Comment by ikkemaniac** [comment]
@GJRos btw, any chance you have a backup of the domoticz executable? that would make be a lot easier? you could then just copy that backed up executable back into the domoticz folder.

**Comment by arbenik** [comment]
@GJRos , I newly updated today to:

Version: 3.8027
Build Hash: 829284b6 

and everything works and pushes fine :) so this was resolved in my opinion.

@ikkemaniac , Thank You very very very much for your help!!!

**Comment by GJRos** [comment]
@ikkemaniac Thanks, it is working.

**Comment by ikkemaniac** [comment]
@GJRos , @arbenik what platform are you guys on and how do you update? Do you just click on the 'check for updates' button in the setting pulldown menu?

**Comment by arbenik** [comment]
@ikkemaniac , I am using Ubuntu 16.04.2 LTS on an off-site VPS and yes, I update by using the options on the Domoticz GUI.

**Comment by ikkemaniac** [comment]
@arbenik ok, so you're on *nix as well. I'm running into problems when trying to update via the web gui (tried if for the first time ever when debugging this issue).

Could you do me a favour and post you log from when you updated? I think you should have at least the following in your log:

```
2017-07-02 17:02:37.311  Starting Upgrade progress...
2017-07-02 17:02:37.312  Starting: domoticz/scripts/download_update.sh domoticz/ /beta
-- maybe something in between those lines --
2017-07-02 17:03:46.416  Executing script: domoticz/scripts/update_domoticz
please standby... (waiting 15 seconds)
```

Please post all lines if possible, tnx!

**Comment by arbenik** [comment]
@ikkemaniac , there were no logs (menu - logs) about the last update. So what I did was downgraded to "updaterelease" through command line, and then next did the beta update through GUI.

The logs were:
```
2017-07-02 19:01:15.269 MySensors: connected to: XXX.XXX.XXX.XXX:XXXX
2017-07-02 19:01:15.270 MySensors: Node: 0, Sketch Name: Sonoff ethernet
2017-07-02 19:01:15.270 MySensors: Node: 0, Sketch Version: 1.0
2017-07-02 19:02:00.797 Starting Upgrade progress...
2017-07-02 19:02:00.799 Starting: /home/smart/domoticz/scripts/download_update.sh /home/smart/domoticz/ /beta
2017-07-02 19:02:03.149 Executing script: /home/smart/domoticz/scripts/update_domoticz
```


And that's all. Don't know if it helps. Tried to find more logs @ /var/log but there were none :(

PS. If you want I can give you access to the VPS and you can see yourself if you can find those logs.

**Comment by arbenik** [comment]
Now, I don't know why but if I don't manually restart the whole vps, I continue to get the "update available" message. I am sure it is a browser thing but I am to lazy to investigate that. Anyway, after the manual restart (reboot now), Domoticz logs continue as follows:

```
2017-07-02 19:01:15.269 MySensors: connected to: XXX.XXX.XXX.XXX:XXXX
2017-07-02 19:01:15.270 MySensors: Node: 0, Sketch Name: Sonoff ethernet
2017-07-02 19:01:15.270 MySensors: Node: 0, Sketch Version: 1.0
2017-07-02 19:02:00.797 Starting Upgrade progress...
2017-07-02 19:02:00.799 Starting: /home/smart/domoticz/scripts/download_update.sh /home/smart/domoticz/ /beta
2017-07-02 19:02:03.149 Executing script: /home/smart/domoticz/scripts/update_domoticz
2017-07-02 19:10:42.051 Domoticz V3.8034 (c)2012-2017 GizMoCuz
2017-07-02 19:10:42.051 Build Hash: a9ad9b67, Date: 2017-07-02 09:50:22
2017-07-02 19:10:42.070 Startup Path: /home/smart/domoticz/
2017-07-02 19:10:42.190 Error: SQL Query("INSERT INTO TimerPlans (ID, Name) VALUES (0, 'default')") : UNIQUE constraint failed: TimerPlans.ID
2017-07-02 19:10:42.195 Error: SQL Query("INSERT INTO TimerPlans (ID, Name) VALUES (1, 'Holiday')") : UNIQUE constraint failed: TimerPlans.ID
2017-07-02 19:10:42.275 Sunrise: 05:13:00 SunSet:20:16:00
2017-07-02 19:10:42.275 EventSystem: reset all events...
2017-07-02 19:10:42.487 PluginSystem: Started, Python version '3.5.2'.
2017-07-02 19:10:42.500 Active notification Subsystems: kodi (1/12)
2017-07-02 19:10:42.509 WebServer(HTTP) started on address: :: with port 8080
2017-07-02 19:10:42.527 Error: WebServer(SSL) startup failed on address 0.0.0.0 with port: 443: use_certificate_chain_file: No such file or directory
2017-07-02 19:10:42.527 Error: WebServer(SSL) check privileges for opening ports below 1024
2017-07-02 19:10:42.528 Proxymanager started.
2017-07-02 19:10:42.530 Starting shared server on: :::6144
2017-07-02 19:10:42.531 RxQueue: queue worker started...
2017-07-02 19:10:42.546 TCPServer: shared server started...
2017-07-02 19:10:42.937 Incoming connection from: XXX.XXX.XXX.XXX
2017-07-02 19:10:44.548 DarkSky: Started...
2017-07-02 19:10:44.549 Hardware Monitor: Started
2017-07-02 19:10:44.553 OpenWeatherMap: Started
2017-07-02 19:10:44.559 EventSystem: reset all events...
2017-07-02 19:10:44.563 EventSystem: reset all device statuses...
2017-07-02 19:10:44.582 Wunderground: Worker started...
2017-07-02 19:10:44.710 Python EventSystem: Module not found - Trying to initialize.
2017-07-02 19:10:44.719 Python EventSystem: Initalizing event module.
2017-07-02 19:10:44.727 EventSystem: Started
2017-07-02 19:10:44.796 PluginSystem: Entering work loop.
2017-07-02 19:10:45.563 MySensors: trying to connect to: XXX.XXX.XXX.XXX:XXXX
2017-07-02 19:10:45.563 MySensors: trying to connect to: XXX.XXX.XXX.XXX:XXXX
2017-07-02 19:10:46.563 MySensors: connected to: XXX.XXX.XXX.XXX:XXXX
2017-07-02 19:10:46.563 MySensors: Gateway Ready...
2017-07-02 19:10:46.563 MySensors: connected to: XXX.XXX.XXX.XXX:XXXX
2017-07-02 19:10:46.563 MySensors: Gateway Ready...
2017-07-02 19:10:46.564 MySensors: Node: 0, Sketch Name: Sonoff ethernet
2017-07-02 19:10:46.564 MySensors: Node: 0, Sketch Version: 1.0
2017-07-02 19:10:47.550 Hardware Monitor: Fetching data (System sensors)
2017-07-02 19:10:47.564 MySensors: Gateway Version: 2.2.0-beta
2017-07-02 19:10:47.565 MySensors: Gateway Version: 2.2.0-beta
2017-07-02 19:10:47.565 MySensors: Gateway Version: 2.2.0-beta
2017-07-02 19:10:47.566 MySensors: Gateway Version: 2.2.0-beta
2017-07-02 19:10:47.579 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:47.595 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.606 (Moti (OWM)) Temp + Humidity + Baro (Moti (OWM) - THB)
2017-07-02 19:10:49.612 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.617 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.622 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.627 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.637 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.637 (Moti (OWM)) Wind (Moti (OWM) - Era)
2017-07-02 19:10:49.642 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.648 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.652 Error: HttpLink: Error sending data to http with POST!
2017-07-02 19:10:49.657 Error: HttpLink: Error sending data to http with POST!
```

...and continuing with the "normal routine" logs of Domoticz.

Hope it helps. Again, if you want to search/test by yourself for logs I can give you access.

**Comment by ikkemaniac** [comment]
@arbenik tnx for the extra effort! It would be interesting to look at some stuff on your machine. How do we get in touch?

**Comment by arbenik** [comment]
@ikkemaniac : drop me a line at: arbennik (at) gmail...

**Comment by gaetan29** [comment]
Hi,
I am running Domoticz 3.8025 and getting similar issue connecting Domtoicz to InfluxDB.
Some data gets transferred (Netatmo temp for exemple), but not RFXCOM temp ...
See log below:

2017-07-08 22:39:17.166 HttpLink: response
2017-07-08 22:39:17.167 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:17.295 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:17.296 HttpLink: response
2017-07-08 22:39:17.298 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:17.444 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:17.444 HttpLink: response
2017-07-08 22:39:17.446 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:17.577 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:17.577 HttpLink: response
2017-07-08 22:39:19.225 HttpLink: sending global variable TELEINFO_CONSO with value: 520.0
2017-07-08 22:39:19.354 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:19.355 HttpLink: response
2017-07-08 22:39:19.355 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:19.489 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:19.489 HttpLink: response
2017-07-08 22:39:19.491 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:19.621 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:19.621 HttpLink: response
2017-07-08 22:39:19.624 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:19.752 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:19.752 HttpLink: response
2017-07-08 22:39:23.628 HttpLink: sending global variable TELEINFO_CONSO with value: 530.0
2017-07-08 22:39:23.763 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:23.763 HttpLink: response
2017-07-08 22:39:23.764 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:23.894 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:23.894 HttpLink: response
2017-07-08 22:39:23.897 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:24.026 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:24.026 HttpLink: response
2017-07-08 22:39:24.029 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:24.159 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:24.160 HttpLink: response
2017-07-08 22:39:25.565 HttpLink: sending global variable TV_CONSO with value: 121.9
2017-07-08 22:39:25.695 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:25.696 HttpLink: response
2017-07-08 22:39:25.698 HttpLink: sending global variable TV_KWH_INSTANTANE with value: 121.900
2017-07-08 22:39:25.829 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:25.829 HttpLink: response
2017-07-08 22:39:25.830 HttpLink: sending global variable TV_KWH_UTILISATION with value: 245080.000
2017-07-08 22:39:25.957 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:25.957 HttpLink: response
2017-07-08 22:39:27.984 HttpLink: sending global variable TELEINFO_CONSO with value: 510.0
2017-07-08 22:39:28.111 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:28.111 HttpLink: response
2017-07-08 22:39:28.112 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:28.242 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:28.242 HttpLink: response
2017-07-08 22:39:28.244 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:28.371 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:28.372 HttpLink: response
2017-07-08 22:39:28.374 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:28.507 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:28.507 HttpLink: response
2017-07-08 22:39:30.389 HttpLink: sending global variable TELEINFO_CONSO with value: 520.0
2017-07-08 22:39:30.521 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:30.521 HttpLink: response
2017-07-08 22:39:30.522 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:30.650 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:30.650 HttpLink: response
2017-07-08 22:39:30.653 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:30.778 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:30.778 HttpLink: response
2017-07-08 22:39:30.781 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:30.920 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:30.920 HttpLink: response
2017-07-08 22:39:41.338 HttpLink: sending global variable TELEINFO_CONSO with value: 530.0
2017-07-08 22:39:41.472 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:41.472 HttpLink: response
2017-07-08 22:39:41.473 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:41.603 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:41.603 HttpLink: response
2017-07-08 22:39:41.606 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:41.734 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:41.734 HttpLink: response
2017-07-08 22:39:41.737 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:41.864 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:41.864 HttpLink: response
2017-07-08 22:39:41.867 HttpLink: sending global variable EAU with value: 36
2017-07-08 22:39:41.996 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:41.996 HttpLink: response
2017-07-08 22:39:41.997 HttpLink: sending global variable EAU with value: 36
2017-07-08 22:39:42.131 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:42.131 HttpLink: response
2017-07-08 22:39:45.706 HttpLink: sending global variable TELEINFO_CONSO with value: 520.0
2017-07-08 22:39:45.835 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:45.835 HttpLink: response
2017-07-08 22:39:45.836 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:45.972 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:45.972 HttpLink: response
2017-07-08 22:39:45.975 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:46.104 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:46.104 HttpLink: response
2017-07-08 22:39:46.107 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:46.276 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:46.276 HttpLink: response
2017-07-08 22:39:47.907 HttpLink: sending global variable TELEINFO_CONSO with value: 530.0
2017-07-08 22:39:48.036 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.036 HttpLink: response
2017-07-08 22:39:48.037 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:48.173 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.173 HttpLink: response
2017-07-08 22:39:48.175 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:48.304 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.304 HttpLink: response
2017-07-08 22:39:48.307 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:48.435 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.435 HttpLink: response
2017-07-08 22:39:48.500 HttpLink: sending global variable TV_CONSO with value: 140.2
2017-07-08 22:39:48.631 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.631 HttpLink: response
2017-07-08 22:39:48.634 HttpLink: sending global variable TV_KWH_INSTANTANE with value: 140.200
2017-07-08 22:39:48.776 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.776 HttpLink: response
2017-07-08 22:39:48.777 HttpLink: sending global variable TV_KWH_UTILISATION with value: 245080.000
2017-07-08 22:39:48.908 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:48.908 HttpLink: response
2017-07-08 22:39:56.707 HttpLink: sending global variable TELEINFO_CONSO with value: 520.0
2017-07-08 22:39:56.841 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:56.841 HttpLink: response
2017-07-08 22:39:56.842 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:56.979 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:56.980 HttpLink: response
2017-07-08 22:39:56.982 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:57.129 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:57.130 HttpLink: response
2017-07-08 22:39:57.132 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:57.271 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:57.271 HttpLink: response
2017-07-08 22:39:58.894 HttpLink: sending global variable TELEINFO_CONSO with value: 530.0
2017-07-08 22:39:59.022 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:59.022 HttpLink: response
2017-07-08 22:39:59.023 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:39:59.150 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:59.151 HttpLink: response
2017-07-08 22:39:59.153 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:39:59.284 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:59.284 HttpLink: response
2017-07-08 22:39:59.287 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:39:59.415 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:39:59.415 HttpLink: response
2017-07-08 22:40:05.673 HttpLink: sending global variable TELEINFO_CONSO with value: 520.0
2017-07-08 22:40:05.855 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:05.855 HttpLink: response
2017-07-08 22:40:05.856 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:40:05.988 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:05.988 HttpLink: response
2017-07-08 22:40:05.991 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:40:06.125 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:06.125 HttpLink: response
2017-07-08 22:40:06.128 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:40:06.258 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:06.258 HttpLink: response
2017-07-08 22:40:07.873 HttpLink: sending global variable TELEINFO_CONSO with value: 530.0
2017-07-08 22:40:08.004 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:08.004 HttpLink: response
2017-07-08 22:40:08.005 HttpLink: sending global variable TELEINFO_LIVRAISONCOURANT with value: 0.0
2017-07-08 22:40:08.136 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:08.136 HttpLink: response
2017-07-08 22:40:08.139 HttpLink: sending global variable TELEINFO_COURANT with value: 2.000
2017-07-08 22:40:08.266 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:08.266 HttpLink: response
2017-07-08 22:40:08.269 HttpLink: sending global variable TELEINFO_PERCENTCHARGE with value: 4.44
2017-07-08 22:40:08.403 Error: HttpLink: Error sending data to http with POST!
2017-07-08 22:40:08.403 HttpLink: response
